.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Bug report
+about: Create a report to help us improve
+
+---
+
+**Expected behavior**
+Describe what you expected to happen.
+
+**Actual behavior**
+Describe what actually happened that was unexpected.
+
+**To Reproduce**
+Steps to reproduce the behavior:
```

Comment:
I would suggest if possible a trimmed down version of Application with exact configuration. (This could be possible next step) but it greatly reduces time to investigate.